src/components/Initiatives/InitiativeDetailsPage/InitiativeMetadataCard/InitiativeMetadataCardUtils.ts

```diff
@@ -23,9 +23,8 @@ export const getTargetDateMessage = (initiative: Initiative) => {
   const remainingDays = daysUntil(initiative.targetDate);
 
   return !isNil(remainingDays) && remainingDays < 0
-    ? `Expired ${moment.utc(initiative.targetDate).local().fromNow()}`
-    : `Due by ${moment
-        .utc(initiative.targetDate)
+    ? `Expired ${moment(initiative.targetDate).local().fromNow()}`
+    : `Due by ${moment(initiative.targetDate)
```

Comment:
Rule of thumb is that our backend returns UTC dates, so we should interpret them as UTC unless otherwise specified. This is diverging from that expectation.

Reply:
Well, in task it says the Brain App on Initiative detail shows correct date, so this change is based on that.
In the service, we cast the date with `moment(targetDate)` in the service and on the page we format that using `initiative.targetDate.local().format('MMMM Do, YYYY')` so it's equivalent to that.

Reply:
We should check to see if the backend is returning the correct date format.

Reply:
BE only returns the date part (`2024-06-30`). If it's UTC, then it's probably wrong everywhere and should be resolved like this `moment.utc(targetDate).format('...')` everywhere to be correct.
cc @cristinabuenahora 
I'll post question to BE eng how they treat the date here.

Reply:
I think we should do whatever we do in Cortex @PavelZurek

Reply:
@cristinabuenahora In that case this PR is correct.
cc @FoodProduct

Reply:
brain-app should be doing exactly what I described ☝️